src/geometry/manifold/algebra/smooth_functions.lean

```diff
@@ -32,6 +32,11 @@ instance has_mul {G : Type*} [has_mul G] [topological_space G] [charted_space H'
   has_mul C^∞⟮I, N; I', G⟯ :=
 ⟨λ f g, ⟨f * g, f.smooth.mul g.smooth⟩⟩
 
+@[simp, to_additive]
+lemma coe_mul {G : Type*} [has_mul G] [topological_space G] [charted_space H' G]
+[has_smooth_mul I' G] (f g : C^∞⟮I, N; I', G⟯) :
+  ⇑(f * g) = f * g := rfl
+
 @[to_additive]
 instance has_one {G : Type*} [monoid G] [topological_space G] [charted_space H' G] :
```

Comment:
Is it worth adding `coe_one : ⇑(1 : C^∞⟮I, N; I', G⟯) = 1` too?

Reply:
Yes, done!